### PR TITLE
ft(S3C-3771): Add support for Utapiv2 client to communicate over TLS

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

**Review by naren-rajendran — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

File: lib/UtapiClient.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ class UtapiClient {
         this.enabledOperationCounters = [];
         this.disableClient = true;
 
-        if (config) {
+        if (config && !config.disableClient) {
```
Comment (anurag4DSB) [inline lib/UtapiClient.js:129]: should it be this.disbleClient?

Comment (tmacro) [reply lib/UtapiClient.js:129]: `config.disableClient` is correct. The intent is to allow config to be passed while not enabling it as cloudserver always instantiates a client at startup and relies on the config being undefined to disable it. Context is that I need to pass the tls certs into the v2 client and would like to keep the logic on cloudservers side simple

Comment (anurag4DSB) [reply lib/UtapiClient.js:129]: Cool

**Pull request merged**